.gitignore

```diff
@@ -0,0 +1,4 @@
+# .gitignore
```

Comment:
- [ ] Try to add test.md file on exception inside this .gitignore file.

Reply:
I have added the test.md file to .gitignore.